numpy/doc/broadcasting.py

```diff
@@ -53,9 +53,10 @@
 2) one of them is 1
 
 If these conditions are not met, a
-``ValueError: frames are not aligned`` exception is thrown, indicating that
-the arrays have incompatible shapes. The size of the resulting array
-is the maximum size along each dimension of the input arrays.
+``ValueError: operands could not be broadcast together`` exception is 
+thrown, indicating that the arrays have incompatible shapes. The size of 
+the resulting array is the maximum size along each dimension of the input 
+arrays.
```

Comment:
nit: could replace `size` with `shape` in this sentence, but not really related to this patch, so feel free to leave as is